### ISSUE TITLE
Add stacked multi preview support for KAPT

### ISSUE THREAD

**Issue by oas004** [airbnb/Showkase#264]
As of #259 we made Showkase support stacking previews. However, because of https://youtrack.jetbrains.com/issue/KT-49682 this was not implemented for KAPT.

Making this issue to remember to add support for this when we can bump to Kotlin version 1.7.20

**Comment by vinaygaba** [comment]
@oas004 We are on a newer version of Kotlin now. Maybe we can revisit this? Hopefully that will help simplify our code a bit and we won't need to fork between kapt and ksp as much as we do.

**Comment by oas004** [comment]
Yeah that sounds like a good idea! I can check this out :)
Do you know if the plan is to continue support for KAPT with Showkase long term? Or should mostly people move to KSP? I think supporting both increases the complexity of this library a lot, so if the long term plan is to only support KSP, I'm not sure about revisiting this case. Otherwise, I will gladly look into this :)

**Comment by oas004** [comment]
Furthermore, from reading though the issue referred to above, If I understand that correctly, this isn't an issue with Kotlin, but with Java annotation processing apis actually. I guess we are using those under the hood from XProcessing lib 🤔 I can try to check if XProcessing lib is supporting repeatable annotations

**Comment by oas004** [comment]
Yeah that makes sense to keep support for it! I can take a look at it :) 
I think if XProcessing lib is supporting repeatable annotations out of the box, then I think that should work. Did you update XProcessing lib as well when moving to the newer Kotlin version? I think if I remember correctly, we have tests that are checking the repeatable annotation feature for KAPT, so I think those would fail if they have support for it? 🤔

**Comment by vinaygaba** [comment]
@oas004 Interesting. Yes the version of xprocessing was upgraded, although it's not the latest version. Was done in this PR - https://github.com/airbnb/Showkase/commit/360259e07d86a28548a2795c2d2d66ffe4f5e80a
